src/test/javascript/spec/component/lecture-unit/attachment-unit/attachment-unit.component.spec.ts

```diff
@@ -85,4 +85,22 @@ describe('AttachmentUnitComponent', () => {
         downloadButton.click();
         expect(downloadFileStub).toHaveBeenCalledTimes(1);
     });
+
+    it('should call completion callback when downloaded', (done) => {
+        attachmentUnitComponent.onCompletion.subscribe((event) => {
+            expect(event.lectureUnit).toEqual(attachmentUnit);
+            expect(event.completed).toEqual(true);
```

Comment:
```suggestion
            expect(event.completed).toBeTrue();
```

Reply:
I really should find a way to have ESlint automatically correct that. 😄